src/index.js

```diff
@@ -204,23 +245,6 @@ export default class HillChart extends EventEmitter {
       .attr('y', calculateTextMarginForY());
   }
 
```

Comment:
What changed here?

Reply:
I ordered the "Render" functions at the same point in the file, `undraggablePoint` got moved up the file

Reply:
You can see it up at line 103 now